app/main.js

```diff
@@ -419,6 +423,40 @@ function getTrayMenu () {
   return Menu.buildFromTemplate(trayMenu)
 }
 
+function updateToolTip () {
```

Comment:
Also here we should do something like `breakPlanner.status()`that would return string for tooltip, not doing it in `main.js`